backend/tests/e2e_tests/__init__.py

```diff
@@ -37,16 +40,28 @@
 }
 
 
+def default_logger(*args, **kwargs):
```

Comment:
may consider name this to `log` or `debug`, `logger` sounds like an object you can call `.info` to log something.